src/Juvix/Core/HR/Parser.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE ExplicitForAll #-}
```

Comment:
To fix formatter error.

Reply:
Hmm odd, did you try adding it to `package.yaml` under `default-extensions:` ?

Reply:
Ah I would try uninstalling `stylish-haskell`, I think the new formatter supersedes it anyways.

Reply:
OK yes I've verified that it is indeed the case. So I removed all the pragmas on top.